Hestia/app/src/main/java/hestia/backend/exceptions/ServerExceptions/ServerException.java

```diff
@@ -0,0 +1,13 @@
+package hestia.backend.exceptions.ServerExceptions;
+
+public abstract class ServerException extends Exception {
```

Comment:
I think the naming should be changed, ServerException and DefinedServerException sound too similar, but they represent completely different problems. DefinedServerException sounds like it is a subclass of ServerException.